src/components/sectionSelector/DropDown.js

```diff
@@ -15,7 +15,7 @@ export default ({
     {label && <label className="inputLabel">{label}</label>}
```

Comment:
```suggestion
    {label && <label className="inputLabel">
    	{label}
	    {required && <span style={{ color: 'red' }}>"*"</span>}
    </label>}
```
Нека сложим звездичка до задължителните полета.